user/decorator.py

```diff
@@ -0,0 +1,26 @@
+import bcrypt
+import jwt 
+
+from django.http           import JsonResponse, HttpResponse
+
+from devx.settings         import SECRET_KEY
+from devx.settings         import ALGORITHM
+from .models               import User
+
+def login_decorator(func):
+    def wrapper(self,request,*args,**kwargs):
```

Comment:
```suggestion
    def wrapper(self, request, *args, **kwargs):
```
파라미터는 간격으로 구분해주셔야 합니다.

Reply:
수정하였습니다.